cdap-api/src/main/java/co/cask/cdap/api/schedule/ScheduleBuilder.java

```diff
@@ -140,8 +145,39 @@
    * @param datasetName the name of the dataset in the specified namespace of the app
    * @param numPartitions the minimum number of new partitions added to the dataset to trigger the schedule
    * @return this {@link ScheduleBuilder}
+   * @deprecated since 4.3.0. Please use {@link ScheduleBuilder#setTriggerOnPartitions(String, String, int)} instead.
```

Comment:
So if we don't deprecate it, do you mean that we should just keep this method then? Won't `triggerOnPartitions` and `setTriggerOnPartitions` both be available to the user then?

Or do you mean we can just remove this method and replace all calls with `setTriggerOnPartitions` since it was never being used yet?